### ISSUE TITLE
Enhance error when validTo is not enough on Limit orders

### ISSUE THREAD

**Issue by elena-zh** [cowprotocol/cowswap#1580]
Related to #1574
Error when insufficient validTo; it would be great to rewrite this text
![rewrite for limit orders.png](https://images.zenhubusercontent.com/5f55ec970a2e890ec10f7778/1e060e51-47c7-4910-815c-9f776439e272)

**Comment by elena-zh** [comment]
The issue will be no longer relevant after #1922 merge
